frontend/src/pages/Common/History/eventTypes.js

```diff
@@ -0,0 +1,33 @@
+import strings from "../../../localizeStrings";
+
+export const getProjectEventTypes = () => [
```

Comment:
I suggest using arrays instead of functions since thinking in terms of "Getters" and "Setters" is not the functional mindset.

Reply:
done